### DISCUSSION TITLE
Add Lama Cleaner info

### DISCUSSION THREAD

**Discussion by Sanster** [AbdBarho/stable-diffusion-webui-docker#239] [Show and tell]
Hi, I am the maintainer of [Lama Cleaner](https://github.com/Sanster/lama-cleaner), it's an image inpainting tool powered by SOTA AI Model. Remove any unwanted object, defect, or people from your pictures or erase and replace(powered by stable diffusion 1.5 inpainting model) anything on your pictures. 

https://user-images.githubusercontent.com/3998421/196976498-ba1ad3ab-fa18-4c55-965f-5c6683141375.mp4

There is already a pre-build docker image [here](https://github.com/Sanster/lama-cleaner#docker), and the Dockerfile is [here](https://github.com/Sanster/lama-cleaner/tree/main/docker).

**Comment by AbdBarho** [comment]
@Sanster this tool is amazing! great work.

I am really unsure why does it need to be added here? it seems that you already have a working docker setup.

Am I missing something?

**Reply by Sanster** [reply]
I may have misunderstood something 😅  and thought this was an AWESOME SD docker list, please ignore this discussion. And your efforts to make these tools more accessible to others are awesome.

**Reply by AbdBarho** [reply]
maybe in the future this repo will turn into a collection of awesome SD tools, in that case, your app definitely be on top!
